build/cmake/package_tester/fdb_c_app/app.c

```diff
@@ -0,0 +1,6 @@
+#define FDB_API_VERSION 610
+#include <foundationdb/fdb_c.h>
+
+int main(int argc, char* argv[]) {
+	return 0;
```

Comment:
I think it would be better to also call at least one function from fdb_c - otherwise you won't catch any linker issues...

Reply:
Done